scripts/f5-config.py

```diff
@@ -14,31 +14,57 @@
 # limitations under the License.
 #
 # (c) 2014, Kevin Carter <kevin.carter@rackspace.com>
+from __future__ import print_function
 import argparse
 import json
 import os
+import sys
 
 import netaddr
 
+IS_LEM = True
 
-PART = 'RPC'
-PREFIX_NAME = 'RPC'
+if 'LAB_DATA_DIR' in os.environ:
+    # If we have LAD_DATA_DIR set, we are operating inside a lab environment
+    LAB_DATA_DIR = os.environ['LAB_DATA_DIR']
+else:
+    print('Not a LEM install, missing LAB_DATA_DIR')
+    IS_LEM = False
+
+
+if 'LAB_NAME' in os.environ:
+    LAB_NAME = os.environ['LAB_NAME'].lower()
+    LAB_NAME_URL = LAB_NAME + ".rpc.rackspace.com"
+else:
+    print('Not a LEM install, missing LAB_NAME')
+    IS_LEM = False
+    LAB_NAME_URL = None
+
+if IS_LEM is True:
+    PART = LAB_NAME.upper()
+    PREFIX_NAME = LAB_NAME.upper()
+else:
+    PART = 'RPC'
+    PREFIX_NAME = 'RPC'
 
 SNAT_POOL = (
     '### CREATE SNATPOOL ###\n'
     'create ltm snatpool /' + PART + '/' + PREFIX_NAME + '_SNATPOOL { members replace-all-with {'
     ' %(snat_pool_addresses)s } }'
 )
 
-#Persistance Profile:
-PERSISTANCE = [
+#  Persistence Profile:
+PERSISTENCE = [
     r'create ltm persistence source-addr /' + PART + '/' + PREFIX_NAME + '_PROF_PERSIST_IP {'
     r' app-service none defaults-from /Common/source_addr'
     r' match-across-services enabled timeout 3600 }',
     r'create ltm persistence cookie /' + PART + '/' + PREFIX_NAME + '_PROF_PERSIST_COOKIE {'
     r' app-service none cookie-name RPC-COOKIE defaults-from /Common/cookie }''\n'
 ]
 
+
+#  TODO: Configure monitoring user
```

Comment:
Does this comment refer to something lab specific? Do we need an issue in our backlog for it? Does this comment need more detail?